### PR TITLE
Issue #TG-465 Nginx changes

### PR DESCRIPTION
**Pull request by rjshrjndrn**
1. trace id propagation
2. Automatic config reload
3. channel, device, app Id logging
4. Content caching

### CODE REVIEW

File: kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml [hunk 1]
```diff
@@ -4,8 +4,6 @@ kind: DaemonSet
 metadata:
   name: nginx-public-ingress
   namespace: {{ .Values.namespace }}
-  annotations:
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml:7]: Why are we removing skater annotation?

File: kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml [hunk 2]
```diff
@@ -5,7 +5,8 @@ metadata:
   name: nginx-public-ingress
   namespace: {{ .Values.namespace }}
   annotations:
-    reloader.stakater.com/auto: "true"
+    # Restarting nginx if ssl cert is changed
+    secret.reloader.stakater.com/reload: "ingress-cert"
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml:9]: Please add merge domain secrets also

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Please add merge domain secrets also

File: kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml (hunk 2 above)
Comment (rjshrjndrn) [reply kubernetes/helm_charts/core/nginx-public-ingress/templates/daemonset.yaml]: Removing the entire block. Nginx reload will take care of the ssl refresh also

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**